### DISCUSSION TITLE
Organisation verified badge with github.io website

### DISCUSSION THREAD

**Discussion by robinpokorny** [community/community#20938] [Other Feature Feedback, Questions, & Ideas]
> To display a "Verified" badge, the website and email information shown on an organization's profile must match the verified domain or domains.
–  [Verifying or approving a domain for your organization](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization)

We would like to get the verified badge for our organization while using a GitHub Pages on the `github.io` subdomain. The domain for our email is verified. However, we cannot (obviously) verify `<org>.github.io`.

We use `<org>.github.io` as our organization's open source home page and while we could move it to `opensource.<org>.com` or similar, we like that it is on GitHub subdomain.

Would it make sense to allow the "Verified" badge when the email domain is verified and the organization has a site on the `<org>.github.io` subdomain, whew `<org>` is the name of the organization on GitHub?

**Comment by thaisonzdx** [comment]
You must own a domain for your organization to get `Verified` bagde, not subdomain. After verified, you can use it or its subdomain for Github Pages. Every subdomain *.github.io is owned by Github so we cannot be verified. Unfortunaly, there is no problem in this policy.

**Reply by robinpokorny** [reply]
@sownt, yes, I understand it the same.

Maybe I should phrase it as a feature request. The problem, IMO, is that it effectively punishes organizations that want to use GitHub features (and promote the GitHub brand as a result) as opposed to handling them on their own.

**Comment by jfullstackdev** [comment]
you can create two Orgs, the other one using the custom domain and the other one using the default one, 

when you create an Org, it's a separate one from other Orgs and the user that created that, so it will not follow the custom subdomain set in another org or the personal account of the Orgs owner

**Reply by robinpokorny** [reply]
Hey! Thanks for responding.

Unfortunately, I do not follow how would that allow me to have either of them verified. Could you elaborate on that?

**Reply by jfullstackdev** [reply]
for example, I have an org `https://github.com/devz-throwable`

if my personal account has the customized domain, then this will not be followed by that org, rather it will remain 

<https://devz-throwable.github.io/>

the same rule applies if you created another org, so you can add custom domain and verify it without affecting the other org

**Reply by robinpokorny** [reply]
To be clear, I want my org (the main one) to have the "verified" badge while listing the `<org>.github.io` as the website. The organization you link to does not have the "verified" badge. I still do not see how having a separate org would help me.
